src/controllers/user.js

```diff
@@ -1,31 +1,28 @@
 import Model from '../models';
 import * as helpers from '../helpers';
 
-const { Users, Followers } = Model;
-const {
-  ApplicationError,
-  NotFoundError,
-  paginator
-} = helpers;
-
+const { Users, Followers, Categories } = Model;
+const { ApplicationError, NotFoundError, paginator } = helpers;
 
 export default {
   /**
-* Generic update profile
-* @function
-*
-* @param {Object} request - express request object
-* @param {Object} response - express response object
-*
-* @returns {Object} - callback that execute the controller
-*/
+   * Generic update profile
```

Comment:
I know I told you guys not to use descriptions like this, in the codebase. It's not really nice still seeing stuff like this. I know it's not your code but could you please help update it? Thank you